src/recipe/error.rs

```diff
@@ -160,6 +188,10 @@ impl fmt::Display for ErrorKind {
             ErrorKind::IfSelectorMissingThen => {
                 write!(f, "Missing `then` field in the `if` selector.")
             }
+            ErrorKind::InvalidMd5 => write!(f, "Invalid MD5 checksum."),
+            ErrorKind::InvalidSha256 => write!(f, "Invalid SHA256 checksum."),
+            ErrorKind::InvalidField(s) => write!(f, "Invalid field `{s}`."),
+            ErrorKind::MissingField(s) => write!(f, "Missing field `{s}`"),
```

Comment:
I think its good practice to start error messages lowercase. This is more correct when you concatenate it to another string like:

```
ParseError: invalid md5 checksum.
```